dbt/runner.py

```diff
@@ -385,25 +387,36 @@ def __init__(self, project, target_path, graph_type, args):
         self.graph_type = graph_type
         self.args = args
 
-        self.target = dbt.targets.get_target(
-            self.project.run_environment(),
-            self.args.threads)
+        profile = self.project.run_environment()
+
+        # TODO validate the number of threads
```

Comment:
Yeah, we can hold off on that. Should be easy enough to add once we get this merged